activities/FractionBounce.activity/js/activity.js

```diff
@@ -6,6 +6,9 @@ requirejs.config({
 	}
 });
 
+var score = 0;
```

Comment:
It's better to avoid global variables. You could use data variables instead.

Reply:
Ok i will make the required changes